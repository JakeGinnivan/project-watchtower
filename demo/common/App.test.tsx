import * as React from 'react'
import { mount } from 'enzyme'
import App from './App'

describe('demo suite', () => {

    it('demo test', () => {
        expect(App).toBeTruthy()
    })

    it('environment variables have to be loaded', () => {
        expect(process.env.PORT).toBeDefined()
    })

    it('snapshot', () => {
        const mounted = mount(<App />)
        expect(mounted).toMatchSnapshot()
    })

})
