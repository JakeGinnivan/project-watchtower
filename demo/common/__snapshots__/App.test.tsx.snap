// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`demo suite snapshot 1`] = `
<App>
  <h1>
    Hello world
  </h1>
</App>
`;
